decidim-core/db/seeds.rb

```diff
@@ -17,7 +17,7 @@
     end,
     homepage_image: File.new(File.join(__dir__, "seeds", "homepage_image.jpg")),
     default_locale: I18n.default_locale,
-    available_locales: Decidim.available_locales,
+    available_locales: [:en, :ca, :es],
```

Comment:
No problem! 😄  I think I am going to change our app generator to include those locales as default because if we leave `Decidim.available_locales` commented it will use all the locales.